### PR TITLE
Update datasource link year display

### PR DESCRIPTION
**Pull request by kylefarmer85**
[ticket](https://trello.com/c/44JRhrRO/147-user-can-view-the-year-of-the-datasource-link-before-clicking-into-it)

Created a new component for the data source link for RightHandMenu. Added switch statements to determine what text should be displayed as the link. The links still don't actually go anywhere.

### CODE REVIEW

File: src/components/RightHandMenu/LinkToSource/LinkToSource.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.rt__link {
```
Comment (daleinen7) [inline src/components/RightHandMenu/LinkToSource/LinkToSource.css:1]: Looks like on the trello ticket image, the link is underlined.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
It's a little hard to tell what's happening in the RightHandMenu component because it looks like every line was reformatted. But assuming the only thing you added to that page was the LinkToSource component that and everything else looks good except a minor styling fix.

File: src/components/RightHandMenu/LinkToSource/LinkToSource.css (hunk 1 above)
Comment (kylefarmer85) [reply src/components/RightHandMenu/LinkToSource/LinkToSource.css:1]: Added!

**Comment by MangeFre** [comment]
LGTM

**Review by daleinen7 — APPROVED** [review]
Looks good!

**Pull request merged**